src/status_im2/contexts/chat/messages/content/view.cljs

```diff
@@ -114,7 +114,7 @@
                                      (js/setTimeout #(reset! show-delivery-state? false)
                                                     delivery-state-showing-time-ms))))
           :on-long-press       #(on-long-press message-data context)}
-         [rn/view {:style {:padding-vertical 8}}
+         [rn/view {:style {:padding-vertical 4}}
```

Comment:
Can you please verify these visual changes using iPhone 11 Pro (or iPhone 12 Pro) and Figma? Let me know if you need any help with this 👍

Reply:
I only have 13 Pro currently, I tested on 7A.
Will post a screen shot for how it looks on 13 Pro

Reply:
![image](https://github.com/status-im/status-mobile/assets/33176106/97f43452-f68d-47d6-8ba1-7f2e9f0a2d8b)